### ISSUE TITLE
Path Variables in terraform commands is not substiting the correct variable ( path: ${CIRCLE_BRANCH} )

### ISSUE THREAD

**Issue by augustine-urolime** [CircleCI-Public/terraform-orb#62]
If we are adding the path as variable the orb fails

Path does not exist: ${CIRCLE_BRANCH}

The code which I am using is

```
version: 2.1
orbs:
  docker: circleci/docker@2.0.3
  terraform: circleci/terraform@3.0.1

jobs:
  terraforming:
    executor: terraform/default
    steps:
      - checkout
      - terraform/init:
          path: ${CIRCLE_BRANCH}
```

**Comment by augustine-urolime** [comment]
Found the solution, so closing the same.

```
version: 2.1
orbs:
  docker: circleci/docker@2.0.3
  terraform: circleci/terraform@3.0.1

jobs:
  terraforming:
    executor: terraform/default
    steps:
      - checkout
      - terraform/init:
          path: << pipeline.git.branch >>

workflows:
  terraform-init:
    jobs:
      - terraforming
```

**Comment by augustine-urolime** [comment]
Resolved.

**Comment by KyleTryon** [comment]
Hello just wanted to share that your original request has been resolved in 3.1.0
